src/tasks/installers/olm.ts

```diff
@@ -321,7 +331,8 @@ export class OLMTasks {
       {
         title: `Check if operator is installed in ${DEFAULT_OPENSHIFT_OPERATORS_NS_NAME} namespace`,
         task: async (ctx: any, task: any) => {
-          if (await kube.operatorSubscriptionExists(SUBSCRIPTION_NAME, DEFAULT_OPENSHIFT_OPERATORS_NS_NAME)) {
+          ctx.subscriptionName = await che.findCheSubscriptionName(flags.chenamespace) || DEFAULT_CHE_OPERATOR_SUBSCRIPTION_NAME
```

Comment:
Is it even possible? Why do we need to check `DEFAULT_CHE_OPERATOR_SUBSCRIPTION_NAME` if `findCheSubscription` returned nothing

Reply:
We need to check it because k8s client fails if one of the args is undefined. It could be anything, actually.

Reply:
I'll add task condition